app/gui/src/project-view/components/ExtendedMenu.vue

```diff
@@ -3,9 +3,13 @@ import ActionMenu from '@/components/ActionMenu.vue'
 import DropdownMenu from '@/components/DropdownMenu.vue'
 import SvgIcon from '@/components/SvgIcon.vue'
 import { type DisplayableActionName } from '@/providers/action'
+import ZoomControl from './ZoomControl.vue'
 
 const open = defineModel<boolean>('open', { default: false })
-const { actions } = defineProps<{ actions: DisplayableActionName[] }>()
+const { actions } = defineProps<{
```

Comment:
Since `actions` and `zoomControls` are used similarly, let's destructure both or neither

```suggestion
defineProps<{
```